File: Code_Python/Exercicio-10/x-pessoas-pedi-sua-idade.py
#questão 15
pessoas = int(input("Digite um número de pessoas que vão participar de um teste: "))
soma = 0
x = 0
if pessoas <= 0:
    print("[Alerta: Número inválido.]")
    pessoas = int(input("Digite um número acima de 0: "))
if pessoas > 0:
    for x in range(pessoas):
        idade = int(input(f"[Pessoa {x+1}] | Digite sua idade: "))
        soma = soma + idade
    media = soma/pessoas
    if 0 <= media <= 25:
        print("A média das idades foi classificada na categoria Jovem.")
    elif 26 <= media <= 60:
        print("A média das idades foi classificada na categoria Adulto.")
    elif media > 60:
        print("A média das idades foi classificada na categoria idosa.")

File: Code_C/Exercicio-08/x-pessoas-pedi-sua-idade.c
#include <stdio.h>
#include <stdlib.h>

void pausar(){
	printf("\nPressione alguma tecla para continuar...");
	getch();
}

int main(){
	int pessoas, idade, media, soma = 0, x = 0;

	printf("Digite um numero de pessoas que vao participar de um teste: ");
	scanf("%d", &pessoas);
	if (pessoas <= 0){
		printf("[Alerta: Numero invalido.] \n");
	}
	
	printf("Digite um numero acima de 0: ");
	scanf("%d", &pessoas);
	if (pessoas > 0){
		for (x = 1; x <= pessoas; x++){
			printf("[Pessoa %d ] | Digite sua idade: ", x);
			scanf("%d", &idade);
			soma = soma + idade;
		}
		media = (soma / pessoas);
		if ((media >= 0) && (media <= 25)){
			printf("A media das idades foi classificada na categoria Jovem.\n");
		}else if ((media >= 26) && (media <= 60)){
			printf("A media das idades foi classificada na categoria Adulto.\n");
		}else if (media > 60){
			printf("A media das idades foi classificada na categoria Idosa.\n");
		}
	}

	pausar();
	return (0);
}
